sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Customized/Models/BackupInstancePolicyInfo.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
+    /// <summary> Policy Info in backupInstance. </summary>
+    public partial class BackupInstancePolicyInfo
+    {
+        /// <summary>
+        /// Gets or sets the DataStore Parameters
+        /// Please note <see cref="DataStoreSettings"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
+        /// The available derived classes include <see cref="OperationalDataStoreSettings"/>.
+        /// Please note <see cref="DataStoreSettings"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
+        /// The available derived classes include <see cref="OperationalDataStoreSettings"/>.
+        /// </summary>
+        [Obsolete("DataStoreParametersList is obsolete and will be removed in a future release. Please do not use it any longer.", false)]
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IList<DataStoreSettings> DataStoreParametersList
```

Comment:
When polishing the API, I realized that we might have an issue in single property flatten for collection type, created [an issue](https://github.com/Azure/autorest.csharp/issues/3058) to track.